web/src/compose_state.ts

```diff
@@ -187,7 +196,7 @@ export function private_message_recipient(value?: string): string | undefined {
 }
 
 export function has_message_content(): boolean {
-    return message_content() !== "";
+    return message_content() !== "" && !is_restored_message_is_unedited();
 }
```

Comment:
I think it's probably safer to have two functions: `has_message_content` and `has_novel_message_content`; one can easily imagine some code that cares about just whether the compose box is empty being unhappy to have the function not do what it says, even if all existing callers want `has_novel_message_content`.

Reply:
Great point, I actually found two callers where it made more sense (I think) to only check `has_message_content`. Would love if you could double check those choices for me though.